lib/addresses.js

```diff
@@ -38,6 +38,9 @@ AddressController.prototype.show = function(req, res) {
 
   this._address.getAddressSummary(req.addr, options, function(err, data) {
 
+    if (data.addrStr)
+      data.addrStr = self.common.translateOutputAddress(data.addrStr);
+
     if(err) {
```

Comment:
if there is an err, then data will not be set, most likely, this could be problematic.

Reply:
Good catch! Fixing.